client/client.go

```diff
@@ -35,10 +35,14 @@ import (
 	"github.com/snapcore/snapd/dirs"
 )
 
-func unixDialer(_, _ string) (net.Conn, error) {
+func privateUnixDialer(_, _ string) (net.Conn, error) {
```

Comment:
> which seems to imply that snapd uses the snapd.socket, and snap uses the snap.socket, which obviously isn't the case.

Well, this sounds like exactly the case to me. Perhaps that's where the disagreement comes from?  We are creating a socket for snaps to talk to snapd routinely, without any special permission. The other socket is a complete management socket for snapd as a whole (including /v2/snapctl), so the bare name seems fine. "snapd-snap" is the socket for a snap to talk to snapd. "Config.InSnap" means the communication with snapd is happening from within a snap.

>  where from within the snap one was available to me, the other wasn't (public and private)

I don't think we share a common meaning for the "private" word. :-)  How is it private? Every single snap will see this socket. Every single user in the system will see this socket. World readable, and writable. Hard for me to empathize with the term.

> snapd-snap.socket might be an improvement, though I still can't look at it and derive meaning from it. Do you like that better?

Yeah, let's rename that way then. Seems more clear indeed.